WalletWasabi/JsonConverters/BitcoinEncryptedSecretNoECJsonConverter.cs

```diff
@@ -15,6 +15,13 @@ public override bool CanConvert(Type objectType)
 		/// <inheritdoc />
 		public override object ReadJson(JsonReader reader, Type objectType, object existingValue, JsonSerializer serializer)
 		{
+			if (reader.Value is string s)
```

Comment:
IMO this could be simplified as follow:
```c#
var value = reader.Value as string;
if(string.IsNullOrWhiteSpace(value))
{
     return null;
}
```
In this way the next if statement is also redundant.

```